mlos_bench/mlos_bench/environments/base_environment.py

```diff
@@ -289,9 +289,10 @@ def _combine_tunables(self, tunables: TunableGroups) -> Dict[str, TunableValue]:
         params : Dict[str, Union[int, float, str]]
             Free-format dictionary that contains the new environment configuration.
         """
-        return tunables.get_param_values(
-            group_names=list(self._tunable_params.get_covariant_group_names()),
-            into_params=self._const_args.copy())
+        params = self._const_args.copy()
+        params.update(tunables.get_param_values(
+            group_names=self._tunable_params.get_covariant_group_names()))
+        return params
```

Comment:
TODO: revert this change - no longer necessary (though this was the only call for that method and might be worth dropping anyways)